src/shims/unix/fs.rs

```diff
@@ -856,7 +852,7 @@ pub trait EvalContextExt<'mir, 'tcx: 'mir>: crate::MiriInterpCxExt<'mir, 'tcx> {
             if let Some(file_descriptor) = this.machine.file_handler.handles.get_mut(&fd) {
                 let result = file_descriptor
                     .seek(communicate, seek_from)?
-                    .map(|offset| i64::try_from(offset).unwrap());
```

Comment:
I actually liked the explicit type here, makes it a bit easier to follow what happens. So if `i64::try_from` still works please keep it.